luigi/worker.py

```diff
@@ -625,6 +626,10 @@ def _generate_worker_info(self):
             pass
         return args
 
+    def _generate_worker_id(self, worker_info):
+        worker_info_str = ', '.join(['{}={}'.format(k, v) for k, v in worker_info])
```

Comment:
I don't think `format` looks any less uglier. f-strings would be a different matter but i don't think they're supported on older python.

Reply:
We promote `format` since it's more pythonic. It's implemented without magic unlike `%`. Luigi only supports 2.7+ (other versions are long deprecated)